heron/api/src/scala/com/twitter/heron/streamlet/scala/converter/ScalaToJavaConverter.scala

```diff
@@ -42,4 +60,21 @@ object ScalaToJavaConverter {
     }
   }
 
+  def toSerializableTransformer[R, T](
+      transformer: SerializableTransformer[R, _ <: T])
+    : com.twitter.heron.streamlet.SerializableTransformer[R, _ <: T] = {
```

Comment:
no need for the full path

Reply:
@nwangtw Thanks for the review. Scala API has own `SerializabLeTransformer` and `Sink` Traits. They need to be transformed to their Java versions before passing to Java API. This class covers this transformation functions and same named classes living in different packages can not use import keyword in same class. So, Scala versions use `import` keyword and java versions use `package name` directly. Same comment is for others, too.

Reply:
@nwangtw Thanks for the comment. Yep, alias make sense and addressed with new commit.